scripts/notify.py

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env python3
+
+## usage:
+# ./notify.py FAILURE
+# ./notify.py SUCCESS
+
+import os
+import json
+import sys
+import requests
+import datetime
+from logsnag import LogSnag
+import slack_notifications as slack
+
+
+#the 4 notification channels:
+# - slack
+# - notifi (works for both phone and mac)
+# - push on the phone (via 'logsnag', pushme disappeared)
+# - telegram (wip)
+
+
+token_file = f'{os.path.realpath(os.path.dirname(__file__))}/../secrets/tokens.json'
```

Comment:
In the bash file we have an example of this file, do we want that here as well? 

```
# example of tokens.json:
# {
#   "slack": "123",
#   "notifi": "456",
#   "pushme": "789"
# }
```

Personally I'm a bit on the fence since it is quite a logical setup for the file